managed-databases/postgresql-and-mysql/how-to/upgrade-version.mdx

```diff
@@ -14,36 +14,39 @@ categories:
+If your database engine is outdated, you can upgrade the version to the latest one anytime via the Scaleway console.
+
+<Message tyepe="note>
+  We recommend that you upgrade your engine version if your Database Instance uses an End of Life engine version that is no longer maintained.
+</Message>
+
+  <Message type="important">
+    The version upgrade is only available for:
+      - Database Instances using outdated engine versions.
+      - Database Instances using Block volumes. If your Database Instance uses a local volume, you must first [change the volume type](/managed-databases/postgresql-and-mysql/how-to/change-volume-type) to a Block volume and try again.
+  </Message>
 
 <Macro id="iam-requirements" />
 
 <Message type="requirement">
   - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
-  - You have a [PostgreSQL or MySQL Database Instance](/managed-databases/postgresql-and-mysql/how-to/create-a-database) running on an End of Life engine version
+  - You have a [PostgreSQL or MySQL Database Instance](/managed-databases/postgresql-and-mysql/how-to/create-a-database) running on an outdated engine version
 </Message>
 
 1. Click **PostgreSQL and MySQL** under **Managed Databases** on the side menu. A list of your Database Instances displays.
+2. Click the name of the database whose engine you want to upgrade. The Database Instance information page appears.
+3. Click **Upgrade** under **Database engine**. A pop-up appears.
+    <Lightbox src="scaleway-engine-version-upgrade-pop-up.webp" alt="The pop-up says: This will upgrade your Database Instance rdb-peaceful-dijkstra.The upgrade will create a clone of your current Database Instance with an up-to-date engine version. Both the original and the new Database Instances will be billed. There is a drop-down to select a new engine version and they type of upgrade: 'Upgrade Only' or 'Upgrade and switch incoming traffic'. In the image 'Upgrade only is selected', and an information box says: 'No impact to your original Database Instance. Your Database Instance will remain available with its original endpoints. If you update your endpoints manually, you can revert to the old ones anytime. " />
+3. Select the upgrade method. Two methods are available:
+    - **Upgrade only**: This method has no impact on your original Database Instance. Your Database Instance will remain available with its original endpoint and will continue to be billed.
+    - **Upgrade and switch incoming traffic**: With this method your endpoint is migrated to the new Database Instance automatically. This option will create a clone of your Database Instance, and will automatically migrate the endpoint to the new Instance. The original Database Instance remains available, and will continue to be billed. The endpoint will be deleted from the original Database Instance.
 
```

Comment:
We must add an info here as it is not indicated. 

When the client choose to Upgrade its instance, there is no synchronisation between the source instance and the new upgraded instance. If the client to not want to loose data, the client application have to stop writting operation while the upgrade procedure is in progress. Otherwise data will still be stored in the old database and the new upgraded one won't have those new data